docs/UserGuide.md

```diff
@@ -256,10 +263,16 @@ Here is an example with steps to follow:
 
 ### Editing your assignment : `edit`
 
-Format: `edit INDEX PREFIX/EDITTED_FIELD`
+Format: `edit INDEX PREFIX/EDITTED_FIELD [MORE_PREFIX/EDITTED_FIELD]`
 
 You can specify an assignment's index to edit its fields. These fields are the name, module code and deadline of assignment.
 
+<div markdown="span" class="alert alert-success">
+
+**:bulb: Tip:**
+You can edit more than one field of an assignment at a time! For example, you can key in `edit 1 n/Tutorial mod/CS2103T` to edit both the name and module code in a single command.
+</div>
```

Comment:
Nice addition :) You may want to further expand on the user benefit as well as Dr Wee mentioned that we should answer the "so that" question. 

A suggestion would be "...so that you can easily amend any mistakes you made when adding assignments".